.babelrc

```diff
@@ -1,5 +1,5 @@
 {
-  "sourceMaps": "true",
-  "stage": 2
-}
-
+  "sourceMaps": true,
+  "stage": 2,
+  "optional": ["runtime"]
```

Comment:
I thought I had installed the babel-runtime package and done this... hmmm... thanks for making sure it's there.